src/app/manage-projects/manage-single-project/manage-single-project.component.ts

```diff
@@ -37,11 +38,18 @@ export class ManageSingleProjectComponent implements OnInit {
                 private http: BackendHttpClient,
                 private manageProjectsService: ManageProjectsService,
                 private route: ActivatedRoute,
-                private broadService: BroadcastService) {
+                private broadService: BroadcastService,
+                private mapService: LocationMapService) {
     }
 
     ngOnInit() {
         this.fetchAllData();
+        setTimeout(() => {
+            this.mapService.getMapViewOnEdit(this.project.location.lat, this.project.location.long);
+        }, 2000);
```

Comment:
Again, this is a hack. It would be better to catch the map event that signalizes that it's initialized, and then do something.